src/components/entity/list.vue

```diff
@@ -13,6 +13,17 @@ except according to the terms contained in the LICENSE file.
   <div id="entity-list">
     <div id="entity-list-actions" class="table-actions-bar">
       <form class="form-inline" @submit.prevent>
+        <div class="form-group">
+          <span class="icon-filter"></span>
+        </div>
+         <label class="form-group">
```

Comment:
I don't think this should be a `<label>` if the element doesn't contain label text. Usually the `.form-group` contains a `.form-label` (e.g., in the `FormGroup` component), and in that context, it makes sense for `.form-group` to be a `<label>`. But given that there isn't label text here, I think this should just be a `<div>`. Then we can add an accessible label to the `<input>` using `aria-label`. To summarize, I think we should:

- Change `<label>` to `<div>`.
- Add `:aria-label="$t('common.search')"` to the `<input>`.